resources/php_ini.rb

```diff
@@ -0,0 +1,16 @@
+resource_name :php_ini
+
+default_action :create
+
+property :mode, String, default: '0644'
+property :options, Hash, default: {}, required: true
+property :path, String, name_property: true
```

Comment:
Let's not use ``path`` and instead just use the default name ``name`` which you don't need to add here.